.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Contributing to AutoPas
+
+**Thanks for contributing to AutoPas!** 
+
+Please keep in mind the following notes while working.
+
+## C++
+### General Notes
+* Cpp standard: C++14. If there is a piece of code, which could be done better using a newer standard, please add a comment like `@todo C++17` including the alternative version of the code. For now, we are stuck with C++14 due to the CUDA dependency.
+* Pointers: Unless there is no special reason, use smart pointers instead of raw pointers.
+* OpenMP: Use AutoPas wrapper functions for OpenMP (`src/autopas/utils/WrapOpenMP.h`) instead of OpenMP functions to allow building without enabled OpenMP.
+* `#pragma once` instead of header guards.
+* `constexpr` instead of `#define`.
+
+### Code Style
+* Private attributes are prefixed with `_`.
+* Every (abstract) class gets its own file, named exactly like the class.
+* Class names start with a capital letter.
+* Use camelCase over snake_case.
+* Google code style is enforced by the CI server.
+* Clang format version 6.0 (Other versions might format slightly differently).
+* Use `make clangformat` before submitting a PR.
+
+### Comment Style
+* Please write full sentences starting with a capital letter and ending with a period.
+* Doxygen (VERSION) is used in this project to create a documentation.
+* Documentation style is Javadoc style.
+* All public methods and attributes need to be documented.
+* The first comment inside a comment block (`/** <comment> */`) is automatically treated as a brief comment and needs to end with a period. The `brief` keyword is omitted (please delete occurrences).
```

Comment:
this is default for doxygen, why mention it here, again?

Reply:
because apparently, some people like to use it and we found multiple occurrences of `@brief` which we would like to avoid in the future.